iam.tf

```diff
@@ -4,29 +4,82 @@
 resource "google_service_account" "gitlab_service_account" {
```

Comment:
Please divide the file to the following sections:
- SA
- Roles
- Bindings

Reply:
https://github.com/SygniaLabs/ScallOps/commit/68b25391f48853d3f35a286fffd747d6fdf0e15d#diff-2bb47ab35b4fcfacd8f871bd1f4bd594f5d1c2a89c0eb861c0e305175a0ef4ec